packages/ai/src/chains/template-generator/index.ts

```diff
@@ -0,0 +1,108 @@
+import { z } from "zod";
```

Comment:
I would not put so much code into index.ts. It's better use as reexport or not use at all because very hard to search navigate.

Reply:
Or just chain.ts

Reply:
we use index only for exporting, nothing else, no logic whatsoever

Reply:
it removes the problem of having many index files open